metron-interface/metron-alerts/src/app/pcap/pcap-packet-line/pcap-packet-line.component.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, OnInit, Input } from '@angular/core';
+import { PdmlPacket, PdmlProto, PdmlField } from '../model/pdml'
+
+@Component({
+  selector: '[app-pcap-packet-line]',
+  templateUrl: './pcap-packet-line.component.html',
+  styleUrls: ['./pcap-packet-line.component.scss']
+})
+export class PcapPacketLineComponent implements OnInit {
+
+  @Input() packet: PdmlPacket
+
+  ip: {
+    timestamp: PdmlField,
+    ip_src_addr: PdmlField, ip_src_port: PdmlField,
+    ip_dest_addr: PdmlField, ip_dest_port: PdmlField,
+    protocol: PdmlField
+  }
+
+  constructor() { }
+
+  ngOnInit() {
+    let gen_proto: PdmlProto = this.packet.protos.filter(p => p.name == "geninfo")[0]
```

Comment:
These variables do not match the style used across Metron in Typescript files.  We generally follow the [Angular](https://angular.io/guide/styleguide) style guide because this is an Angular application but I'm not sure we have officially agreed on that as a community in a discussion thread.  I will get that going.

I'm fairly confident that this won't match whatever style we choose so I would suggest we make it match our current codebase in the meantime.

Reply:
Thanks @merrimanr! I addressed this in a new commit.